docs/development/core/development-functional-tests.asciidoc

```diff
@@ -61,14 +61,21 @@ export TEST_ES_PASS=<password>
 node scripts/functional_test_runner
 ----------
 
-** Selenium tests can be run in headless mode by setting the environment variable below.  Unset this variable to show the browser.
+** Functional tests are run in headless mode on CI. You can activate headless mode locally by setting the environment variable below:
```

Comment:
- Are all of our functional tests Selenium?  If not, I recommend we switch it back to 'Selenium tests' instead of 'Functional test' at the beginning of the sentence.
- Nice on adding info for CI that it is run in headless mode. 
- I think we should add back that you need to unset headless env variable to show the browser again.  I think sometimes it causes confusion as folks just set it to 0 or , but that won't work since I believe the code looks for the variable to exist or not exist.  At least that is how it used to work, we should verify.

Reply:
>Are all of our functional tests Selenium? If not, I recommend we switch it back to 'Selenium tests' instead of 'Functional test' at the beginning of the sentence.
You are right, Liza. I will revert it back.
>I think we should add back that you need to unset headless env variable to show the browser again. I think sometimes it causes confusion as folks just set it to 0 or , but that won't work since I believe the code looks for the variable to exist or not exist. At least that is how it used to work, we should verify.
There is no need to unset env variable, because it is not set via the runner. When you are testing locally, it is always with the head. My point was to clarify how the user can enable headless if there is a need.
And I agree: currently, it works a bit weird (checking the variable to exist or not exist). Should we fix it in other PR? I'm into it.

Reply:
I think it is important to clarify that on CI and locally we have a different approach. Sometimes there is a need to run tests in headless to see the same error you got on CI.

`
Selenium tests are run in headless mode on CI. Locally the same tests will be executed in a real browser.
You can activate headless mode by setting the environment variable below:
`

@liza-mae will it give more clarification?

Reply:
@dmlemeshko, yes a PR to fix the headless env would be great.  The rest of the doc looks good.  I would recommend a small section on tags but we have now is fine.